src/main/java/org/terasology/logic/inventory/ItemCommands.java

```diff
@@ -114,6 +126,43 @@ public String give(
         return "Could not find an item or block matching \"" + itemPrefabName + "\"";
     }
 
+
+    @Command(shortDescription = "Removes an item from your inventory",
+            helpText = "Removes the desired number of the given item from your inventory",
+            runOnServer = true,
+            requiredPermission = PermissionManager.CHEAT_PERMISSION)
+    public String remove(
+            @Sender EntityRef client,
+            @CommandParam("prefabId") String inventoryObjectId,
+            @CommandParam(value = "quantity", required = false) Integer quantity,
+            @CommandParam(value = "blockShapeName", required = false) String shapeUriParam) {
+
+        int removalQuantity = quantity != null ? quantity : 1;
+        if (removalQuantity < 1) {
+            return "Invalid quantity of items!";
+        }
+
+        Set<ResourceUrn> matches = assetManager.resolve(inventoryObjectId, Prefab.class);
+
+        if (matches.size() == 1) {
+            Prefab prefab = assetManager.getAsset(matches.iterator().next(), Prefab.class).orElse(null);
+            String message = removeItem(prefab, quantity, client);
+            if (message != null) {
+                return message;
+            }
```

Comment:
I don't think we need to do the `null` check and return early. When defining `String message` before the if-else blocks, this should also work:

```suggestion
            message = removeItem(prefab, quantity, client);
```